openrtb_ext/viewability_vendors.go

```diff
@@ -0,0 +1,22 @@
+package openrtb_ext
+
+import "fmt"
+
+var viewabilityVendorMap = map[string]string{
```

Comment:
I think these would be better off in `static/metrics/*.yaml` files, like the adapters have. There are a bunch of advantages to this, which we can talk about if you're curious.

It seems to me like we should be able to answer the following questions using just data stored in the `yaml` files:

1. Which metric types does PBS recognize?
2. Which metric vendors does PBS recognize?
3. Which URLs correspond to each vendor?
4. Which vendors are supported by each adapter? (You're already doing this one ^^)

It seems like there should also be unit tests which ensure that:

1. Every `vendor` which an adapter says it supports is also one recognized by Prebid Server.
2. No two `vendors` have the same URL.